oneanddone/tasks/forms.py

```diff
@@ -25,12 +27,16 @@ def __init__(self, *args, **kwargs):
 
 
 class FeedbackForm(forms.ModelForm):
+    CHOICES = ((1, mark_safe(_('<img src="/static/icons/happy.png" alt="approve" width= "87" height = "87" title="approve">'))),
```

Comment:
`CHOICES` is not a very descriptive name for this variable. There could be other items on the form that require choices. I suggest you use the name of the field (in this case `rating`) as part of the variable name.